system/src/Grav/Console/Cli/InstallCommand.php

```diff
@@ -66,12 +66,11 @@ protected function serve()
         $this->destination = rtrim($this->destination, DS) . DS;
         $this->user_path = $this->destination . USER_PATH;
 
-        if (false === $this->isWindows()) {
-            $local_config_file = exec('eval echo ~/.grav/config');
-            if (file_exists($local_config_file)) {
-                $this->local_config = Yaml::parse($local_config_file);
-                $this->output->writeln('Read local config from <cyan>' . $local_config_file . '</cyan>');
-            }
+        $home_folder = getenv('HOME') ?: getenv('HOMEDRIVE') . getenv('HOMEPATH');
```

Comment:
why not put this (69-73 in this file) and line 116-119 from gmp into the console trait?

```
public function loadLocalConfig($cli=false) 
{
    $home_folder = getenv('HOME') ?: getenv('HOMEDRIVE') . getenv('HOMEPATH');
    $local_config_file = $home_folder . '/.grav/config';
    if (file_exists($local_config_file)) {
        $this->local_config = Yaml::parse($local_config_file);
        if ($cli) {
            $this->output->writeln('Read local config from <cyan>' . $local_config_file . '</cyan>');
        }
    }
}
```

calling it with `$this->loadLocalConfig(true)` or `$this->loadLocalConfig(false)`

or change the ` if ($cli)` to:

```
if ($this->output && get_class($this->output)=='OutputInterface') 
```
and remove the `$cli` parameter from `loadLocalConfig`